src/set.js

```diff
@@ -0,0 +1,99 @@
+export default class Set {
+  constructor() {
+    this.length = 0
+    this.elements = []
+  }
+
+  add = (value) => {
+    if(!this.contains(value)){
+      this.elements.push(value)
+      this.length += 1
+    }
+  }
+
+  isEmpty = () => this.length === 0
+
+  contains = (value) => {
+    for (let i = 0; i < this.length; i++) {
+      if (this.elements[i] === value) {
+        return true
+      }
+    }
+    return false
+  }
+
+  remove = (value) => {
+    if(this.contains(value)){
+      let removeIndex = 0
+      for(let i = 0; i < this.length; i++){
+        if(this.elements[i] === value){
+          removeIndex = i
+        }
+      }
+      this.elements.splice(removeIndex, 1)
+      this.length -= 1
+    }
+  }
+
+  forEach = passedInFunction => {
+    const setSize = new Number(this.length)
```

Comment:
This jumped out at me - why `new Number( this.length )`?  Why not just `this.length()`?